CHANGELOG.md

```diff
@@ -1,19 +1,35 @@
-# Change Log
+# Node.js Agent for Google Cloud Trace ChangeLog
 
-All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
+## 2017-09-12, Version 2.1.4 (Beta), @matthewloring
```

Comment:
This is a semver minor, so it should be 2.2.0.

Reply:
Fixed.